worlds/sc2/docs/setup_fr.md

```diff
@@ -0,0 +1,191 @@
+
+## Logiciels requis
+
+- [*StarCraft 2*](https://starcraft2.com/en-us/)
+- [La version la plus récente d'Archipelago](https://github.com/ArchipelagoMW/Archipelago/releases)
+
+## Comment est-ce que j'installe ce *randomizer*?
+
+1. Installer *StarCraft 2* et Archipelago en suivant les instructions indiquées dans les liens précédents. Le client de 
+*StarCraft 2 Archipelago* est téléchargé par le programme d'installation d'Archipelago.
+   - Les utilisateurs de Linux devraient aussi suivre les instructions qui se retrouvent à la fin de cette page 
+(["Exécuter sous Linux"](#exécuter-sous-linux)).
+2. Exécuter `ArchipelagoStarcraft2Client.exe`.
+   - Uniquement pour cette étape, les utilisateurs de macOS devraient plutôt suivre les instructions qui se trouvent à 
+["Exécuter sous macOS"](#exécuter-sous-macos).
+3. Dans le client de *StarCraft 2 Archipelago*, écrire la commande `/download_data`. Cette commande va lancer 
+l'installation des *Maps* et des *Data files* qui sont nécessaires pour jouer à *StarCraft 2 Archipelago*.
+
+## Où est-ce que j'obtiens le fichier de configuration (i.e., le *yaml*) pour ce jeu?
+
+Un fichier dans le format *yaml* est utilisé pour communiquer à Archipelago comment vous voulez que votre jeu soit 
+*randomized*. 
+Ce dernier est nécessaire même si vous voulez utiliser les options par défaut. 
+L'approche usuelle pour générer un *multiworld* consiste à avoir un fichier *yaml* par monde.
+
+Il y a trois approches pour obtenir un fichier *yaml* pour *StarCraft 2 Randomizer*:
+* Vous pouvez aller à la page [*Player options*](/games/Starcraft%202/player-options) qui vous permet de définir vos 
+choix via une interface graphique et ensuite télécharger le *yaml* correspondant à ces choix.
+* Vous pouvez obtenir le modèle de base en le téléchargeant à la page 
+[*Player options*](/games/Starcraft%202/player-options) ou en cliquant sur *Generate template* après avoir éxécuté le 
+*Launcher* d'Archipelago (i.e., `ArchipelagoLauncher.exe`). Ce modèle de base inclut une description pour chacune des 
+options et vous n'avez qu'à modifier les options dans un éditeur de texte de votre choix.
+* Vous pouvez demander à quelqu'un d'autre de partager un de ces fichiers *yaml* pour l'utiliser ou l'ajuster à vos 
+préférences.
+
+Prenez soin de vous rappeler du nom de joueur que vous avez inscrit dans la page à options ou dans le fichier *yaml* 
+puisque vous en aurez besoin pour vous connecter à votre monde!
+
+Notez que la page *Player options* ne permet pas de définir certaines des options avancées, e.g., l'exclusion de 
+certaines unités ou de leurs améliorations. 
+Utilisez la page [*Weighted Options*](/weighted-options) pour avoir accès à ces dernières.
+
+Si vous désirez des informations et/ou instructions générales sur l'utilisation d'un fichier *yaml* pour Archipelago, 
+veuilliez consutler [*Creating a YAML*](/tutorial/Archipelago/setup/en#creating-a-yaml).
+
+### Questions récurrentes à propos du fichier yaml
```

Comment:
yaml en plus de formatting? (only for consistency)

Reply:
the others are italicized (I gave up on typing in French as it's too early for my brain atm)

Reply:
True!